sdk/python/tests/components/test_python_op.py

```diff
@@ -348,7 +348,7 @@ def assert_is_none(a, b, arg=None) -> int:
             return 1
 
         func = assert_is_none
-        op = comp.func_to_container_op(func, output_component_file='comp.yaml')
+        op = comp.func_to_container_op(func)
```

Comment:
I think removing this param will make the coverage of the unittest a bit of smaller.
If output_component_file is not specified, component_spec won't be translated into a yaml dump. WDYT?

Reply:
Thanks. Did not think about that. That piece of code would only be a couple of lines, but we might want to test it.

Reply:
I've added two new test that test the writing components to file.